### DISCUSSION TITLE
Reservoirs and Storage Plants

### DISCUSSION THREAD

**Discussion by javierclb** [ANL-CEEESA/UnitCommitment.jl#8] [Feature Requests]
Hi i just wanted to congratulate you for the work yo have done so far, the package is looking very well. 

I was wondering, do you any have plans of including other technologies such as reservoirs and storage? I'm asking because those kinds of plants turn to be crucial in many systems and markets for supplying reserve and sufficiency and both of them have their own complex formulation.

Thank you very much in advance!

**Comment by iSoron** [comment]
Hi @javierclb, thank you for the suggestion. Reservoirs and energy storage are definitely within the scope of the package. I think the first step towards adding these to UC.jl would be collecting publicly available and realistic instances containing storage. If you (or anyone else) is aware of any interesting data sets, please let us know. Contributions are very welcome.
